src/screens/ExchangeModal.tsx

```diff
@@ -62,29 +69,33 @@ import {
 } from '@/raps';
 import {
   swapClearState,
+  TypeSpecificParameters,
   updateSwapSlippage,
   updateSwapTypeDetails,
 } from '@/redux/swap';
 import { ETH_ADDRESS, ethUnits } from '@/references';
 import Routes from '@/navigation/routesNames';
-import styled from '@/styled-thing';
-import { position } from '@/styles';
 import { ethereumUtils, gasUtils } from '@/utils';
 import { useEthUSDPrice } from '@/utils/ethereumUtils';
-import logger from '@/utils/logger';
+import logger from 'logger';
 
 export const DEFAULT_SLIPPAGE_BIPS = {
   [Network.mainnet]: 100,
   [Network.polygon]: 200,
   [Network.optimism]: 200,
   [Network.arbitrum]: 200,
+  [Network.goerli]: 100,
+  [Network.ropsten]: 100,
+  [Network.rinkeby]: 100,
+  [Network.kovan]: 100,
```

Comment:
same for this, it was fremoved https://github.com/rainbow-me/rainbow/pull/4014/files#r960026216, this shouldn't be available anymore for networks different than goerli and mainnet